substrate-ingest/src/parse/block.ts

```diff
@@ -117,6 +118,11 @@ export class BlockParser {
         })
     }
 
+    @def
+    logs(): EvmLog[] {
```

Comment:
Hm... i thought that it would be useful for parquet also
But probably a separate table will slow down querying because of different query engine so i'll move this to the sink